connect/runtime/src/test/java/org/apache/kafka/connect/storage/OffsetStorageWriterTest.java

```diff
@@ -16,23 +16,36 @@
  */
 package org.apache.kafka.connect.storage;
 
+import org.apache.kafka.clients.producer.MockProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.utils.MockTime;
 import org.apache.kafka.connect.util.Callback;
+import org.apache.kafka.connect.util.KafkaBasedLog;
+import org.apache.kafka.connect.util.LoggingContext;
+import org.apache.kafka.connect.util.TopicAdmin;
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.ArgumentCaptor;
+import org.powermock.reflect.Whitebox;
```

Comment:
We shouldn't be using EasyMock / PowerMock anymore (https://issues.apache.org/jira/browse/KAFKA-7438). Previous usages in this test class were removed in https://github.com/apache/kafka/pull/11450